matcher/service/openreview_interface.py

```diff
@@ -90,15 +90,17 @@ def get_all_edges(client, edge_invitation_id, logger=None):
     logger.debug('GET grouped edges invitation id={}'.format(edge_invitation_id))
     for group in edges_grouped_by_paper:
         forum_id = group['id']['head']
-        for group_value in group['values']:
-            all_edges.append(build_edge(
-                edge_invitation,
-                forum_id,
-                group_value['tail'],
-                group_value.get('weight'),
-                group_value.get('label'),
-                None
-            ))
+        if forum_id in all_papers:
```

Comment:
this if can be too expensive, you can build a dictionary and check by the key. 
Instead of doing an if in each loop you can apply a filter after you get the edges.

Reply:
Fixed. @melisabok Please check if this is what you had in mind.

Reply:
Looks good!

I made a few comments.